apps/etterlatte-pdltjenester/src/main/kotlin/Application.kt

```diff
@@ -25,6 +29,10 @@ class Server(applicationContext: ApplicationContext) {
             applicationConfig = applicationContext.config,
         ) {
             personRoute(applicationContext.personService)
+
+            route("/api/pdltjenester") {
+                personWebRoute(applicationContext.personWebService, SporingService(Sporingslogg()))
```

Comment:
Tror du trenger en `logback-sporing.xml` tilsvarende som i grunnlag for at den treffer riktig logg